assembl/static2/js/app/components/home/header/synthesis.jsx

```diff
@@ -1,54 +1,37 @@
 import React from 'react';
-import { connect } from 'react-redux';
-import { Translate, Localize } from 'react-redux-i18n';
+import { I18n } from 'react-redux-i18n';
+
+import BoxWithHyphen from '../../common/boxWithHyphen';
 import { getConnectedUserId, getDiscussionSlug } from '../../../utils/globalFunctions';
 import { getCurrentView, get, getContextual } from '../../../utils/routeMap';
 
 class Synthesis extends React.Component {
   render() {
-    const { synthesis } = this.props.synthesis;
-    const slug = { slug: getDiscussionSlug() };
-    const connectedUserId = getConnectedUserId();
-    const next = getCurrentView();
-    return (
-      <div className="synthesis-container">
-        {synthesis &&
-          Object.keys(synthesis.lastPublishedSynthesis).length > 0 &&
-          synthesis.lastPublishedSynthesis.introduction &&
-          <a
-            href={
-              connectedUserId
-                ? `${get('oldDebate', slug)}/posts/${encodeURIComponent(synthesis.lastPublishedSynthesis.published_in_post)}`
-                : `${getContextual('login', slug)}?next=${next}`
-            }
-          >
-            <div className="insert-box">
-              <h3 className="dark-title-4 ellipsis">
-                <div>
-                  <Translate value="synthesis.title" />
-                </div>
-                <div className="ellipsis margin-xs">
-                  {synthesis.lastPublishedSynthesis.subject}
-                </div>
-              </h3>
-              <div className="box-hyphen">&nbsp;</div>
-              <div className="date">
-                <Localize value={synthesis.lastPublishedSynthesis.creation_date} dateFormat="date.format2" />
-              </div>
-              <div className="insert-content margin-s">
-                {<p dangerouslySetInnerHTML={{ __html: synthesis.lastPublishedSynthesis.introduction }} />}
-              </div>
-            </div>
-          </a>}
-      </div>
-    );
+    const { error, lastPublishedSynthesis, loading } = this.props.synthesis;
+    if (!loading && !error && lastPublishedSynthesis) {
+      const { subject, introduction, publishedInPost, creationDate } = lastPublishedSynthesis;
+      const title = I18n.t('synthesis.title');
+      const slug = { slug: getDiscussionSlug() };
+      const connectedUserId = getConnectedUserId();
+      const next = getCurrentView();
+      const href = connectedUserId
```

Comment:
No, public debates should allow guest visitors to read a synthesis

Reply:
@swergas Please create an issue in jira.